tests/utils/test_inference.py

```diff
@@ -163,7 +162,7 @@ def test_gpu_huggingface_export_for_inference_onnx():
     import onnxruntime as ort
     import transformers
 
-    from composer.functional import apply_fused_layernorm
```

Comment:
How did this work? There was no apply_fused_layernorm function?

Reply:
It is in the fused_layernorm.py algorithm